src/pages/SignIn/SignIn.jsx

```diff
@@ -11,6 +13,7 @@ const SignIn = () => {
       </Text>
       <div className="flex w-[400px] mx-auto flex-col flex-wrap mb-6 md:mb-0 mt-6 md:mt-2 gap-4">
```

Comment:
`w-[400px]` -> `max-w-[400px]`
and `mb-6 md:mb-0` style makes the gap between the 'continue with email' and 'create account' button changes as the window size changes, so i like the style without that style